### DISCUSSION TITLE
Getting 405 when trying to make a POST request to the com.atproto.server.createSession endpoint

### DISCUSSION THREAD

**Discussion by GV14982** [bluesky-social/atproto#2572] [Dev Debugging] [answered]
Hey all, really love what you are building here. I was starting to explore the API a bit, and I wanted to get an auth token so I could start poking around and maybe find some way to contribute. I created an app password and tried to use the `com.atproto.server.createSession` xrpc method, but I keep getting a `Method Not Allowed` error. Here is my curl request for reference:

```bash
curl -L 'https://public.api.bsky.app/xrpc/com.atproto.server.createSession' \
    -H 'Content-Type: application/json' \
    -H 'Accept: application/json' \
    -d '{"identifier": "gvasquez.dev", "password": "$APP_PASSWORD"}'
```

I can post the error if you want, but it's just HTML with a body of `Method Not Allowed`. Any idea what I might be doing wrong? I also tried directly specifying the request method to POST, but curl infers that because of the use of the `-d` flag.

**Comment by snarfed** [comment, marked as answer]
You need to make that call against your account's PDS, ie either `bsky.social` or the specific PDS [in your DID doc's `serviceEndpoint` field](https://atproto.com/specs/did#did-documents), not against the AppView.

**Reply by GV14982** [reply]
Oh awesome thanks!
